sql/snowflake/SnowflakeParser.g4

```diff
@@ -1282,7 +1282,7 @@ alter_tag
 
 alter_task
     : ALTER TASK if_exists? object_name resume_suspend
-    | ALTER TASK if_exists? object_name ( REMOVE | ADD ) AFTER string_list 
+    | ALTER TASK if_exists? object_name ( REMOVE | ADD ) AFTER string_list
```

Comment:
Please next time avoid fixing unrelated problems in the same MR (excess whitespaces in this case).